frontend/components/commons/results/StatusTag.vue

```diff
@@ -28,25 +21,77 @@ export default {
       type: String,
     },
   },
+  computed: {
+    getStatusInfo() {
+      switch (this.title.toLowerCase()) {
+        case "pending":
+          return {
+            name: "Pending",
+            icon: null,
+            class: "--pending",
+          };
+        case "edited":
+          return {
+            name: "Pending",
+            icon: "time",
+            class: "--edited",
+          };
+        case "discarded":
+          return {
+            name: "Discarded",
+            icon: "discard",
+            class: "--discarded",
+          };
+        case "validated":
+          return {
+            name: "Validate",
+            icon: "validate",
+            class: "--validated",
+          };
+        case "submitted":
+          return {
+            name: "Submitted",
+            icon: "validate",
+            class: "--submitted",
+          };
+        default:
+          console.log(`wrong status: ${this.title}`);
+      }
+    },
```

Comment:
1/ use UPPERCASE
2/ you can use and complete with missing status the RECORD_STATUS object from record.queries.js